tests/test_xsd.py

```diff
@@ -50,17 +58,21 @@ def test_happy_path_mk8s(testdir, properly_decorated_test_function):
 """  # noqa
 
     xml_doc = run_and_parse_with_config(testdir, config).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('mk8s')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('mk8s')))
```

Comment:
Since an external program is accessing this method it should be marked as public.